Source/OxyPlot/Axes/Rendering/HorizontalAndVerticalAxisRenderer.cs

```diff
@@ -248,6 +249,9 @@ protected static void SnapTo(double target, ref double v, double eps = 0.5)
         /// <param name="titlePosition">The title position.</param>
         protected virtual void RenderAxisTitle(Axis axis, double titlePosition)
         {
+            ScreenPoint pt0 = new ScreenPoint(0, 0);
+            ScreenPoint pt1 = new ScreenPoint(0, 0);
```

Comment:
can we move these closer to where they are used, and can we rename them to `areaTopLeft` and `areaBottomRight` or similar?

Reply:
done